hypha/apply/users/forms.py

```diff
@@ -84,6 +84,12 @@ def clean_slack(self):
 
         return slack
 
+    def clean_email(self):
+        email = self.cleaned_data['email']
+        if User.objects.filter(email=email).exists():
```

Comment:
Let's not put any business logic in the `clean_email`, any logic should go the view layer or service layer. when you are assigning `email = self.instance.email` things get very confusing down the line.